libtsuba/include/tsuba/tsuba.h

```diff
@@ -27,7 +30,10 @@ class KATANA_EXPORT RDGFile {
   RDGFile(const RDGFile&) = delete;
   RDGFile& operator=(const RDGFile&) = delete;
 
-  RDGFile(RDGFile&& f) noexcept : handle_(f.handle_) {}
+  RDGFile(RDGFile&& f) noexcept : handle_(f.handle_) {
+    f.handle_ = RDGHandle{nullptr};
```

Comment:
@tylershunt : FYI: I caught a double-deletion bug in this move constructor and fixed it. Seemingly, this wasn't being called before.